inference_rules.adoc

```diff
@@ -468,19 +470,19 @@ Examples of forbidden bus interconnect include: PCIe/CXL/CCIX, Hypertransport, N
 
 Additionally, any interconnect not listed in the permitted list is forbidden unless clearance is first obtained from the MLPerf Inference WG.
 
-The SUT and QDL must run on two physically separated and distinct systems.
+The SUT and QDL must run on two physically separated and distinct systems. The SUT can contain multiple Nodes.
 
 Fabric and protocol must be reported in the submission metadata. Submission metadata must be sufficient to determine OSI layers one through four of the submission's network stack.
 
 ==== SUT over the Network and Setup Constraints
 
 * SUT parameters and configuration must be uniquely and specifically named in the submission results.
-* Everything outside the LoadGen node should be considered as part of the SUT, for instance for counting power and latency. As an example, components outside the nodes like switch or load balancer should be considered part of the SUT. 
+* Everything outside the LoadGen node should be considered as part of the SUT, for instance for counting power and latency. As an example, components outside the nodes like a switch or load balancer should be considered part of the SUT. The SUT can contain multiple Nodes.
```

Comment:
Just for "don't repeat yourself" reasons I suggest we just say that the SUT can contain multiple nodes one time.

Reply:
OK. I will delete it here.